src/helpers/emailConfig.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable require-jsdoc */
+// eslint-disable-next-line import/prefer-default-export
+export function emailConfig({ email, subject, content } = {}) {
+  return {
+    from: process.env.SEND_EMAIL,
```

Comment:
If no problem add `SEND_EMAIL` in env example

Reply:
Done